flink-yarn/src/main/java/org/apache/flink/yarn/Utils.java

```diff
@@ -263,6 +265,13 @@ private static LocalResource registerLocalResource(FileSystem fs, Path remoteRsr
 		return localResource;
 	}
 
+	public static void setAclsFor(ContainerLaunchContext amContainer, org.apache.flink.configuration.Configuration flinkConfig) {
+		amContainer.setApplicationACLs(new HashMap<ApplicationAccessType, String>(){{
```

Comment:
Because double brace initialization [has some caveats](https://stackoverflow.com/a/1958961), I wouldn't use it.